api/tests/core/bookings/test_repository.py

```diff
@@ -2811,3 +2811,37 @@ def test_get_deposit_booking():
         current_deposit_bookings = get_bookings_from_deposit(current_deposit_id)
 
     assert set(current_deposit_bookings) == {current_deposit_booking, current_deposit_booking_2}
+
+
+@pytest.mark.usefixtures("db_session")
+class UnretrievedGoodsTest:
+    def test_find_unretrieved_bookings(self):
+        cannot_expire_product = offers_factories.DigitalProductFactory()
+        cannot_expire_offer = offers_factories.OfferFactory(product=cannot_expire_product)
+        cannot_expire_stock = offers_factories.StockFactory(offer=cannot_expire_offer)
```

Comment:
variante un peu plus courte : 
```suggestion
        cannot_expire_stock = offers_factories.StockFactory(offer=offers_factories.DigitalOfferFactory())
```

(on pourrait peut-être définir une `DigitalStockFactory`, au passage)